lib/web_ui/dev/test_runner.dart

```diff
@@ -240,12 +243,38 @@ class TestCommand extends Command<bool> with ArgUtils {
     }
 
     if (htmlTargets.isNotEmpty) {
-      await _buildTests(targets: htmlTargets, forCanvasKit: false);
+      await _buildTestsInParallel(targets: htmlTargets, forCanvasKit: false);
     }
 
+    // Currently iOS Safari tests are running on simulator, which does not
+    // support canvaskit backend.
```

Comment:
Did you mean to put this comment here? I'm having trouble connecting it to the `if` block below. There's nothing in the code related to iOS, Safari, or simulators.

Reply:
Yes, if anyone is reading the code, hoping to understand how a test runs, I'm telling them "be aware, these lines won't run on iOS Safari"

This is also in the skips in the individual tests but I think it's good to leave these stories to the readers.